torch/_inductor/cudagraph_trees.py

```diff
@@ -711,20 +738,25 @@ def _record(self, model, inputs):
             ]
             check_memory_pool(self.cuda_graphs_pool, memory)
 
-        with torch.cuda.device(self.device), clear_cublas_manager(), torch.cuda.graph(
-            self.graph, stream=self.stream, pool=self.cuda_graphs_pool
-        ):
-            static_outputs = model(inputs)
+        with preserve_rng_state():
```

Comment:
nit: You don't actually need to nest the context managers right? You can chain this one with the rest.